CONTRIBUTING.md

```diff
@@ -46,5 +46,8 @@ docker-compose --project-name django-multitenant up
 Then in another shell run the tests:
 
 ```bash
+DJANGO_VERSION=4.0
+CITUS_VERSION=11.2
+make test-dependencies 
```

Comment:
```
pip install -r /build/requirements/test-requirements.txt
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/build/requirements/test-requirements.txt'
```